standalone-metastore/metastore-server/pom.xml

```diff
@@ -491,6 +491,53 @@
         </plugins>
       </reporting>
     </profile>
+    <profile>
+      <id>dist</id>
+    </profile>
+    <profile>
+      <id>docker</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-metastore</artifactId>
+          <version>${project.version}</version>
+          <scope>runtime</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-standalone-metastore-rest-catalog</artifactId>
```

Comment:
when I use the command `mvn clean install -DskipTests -Pdocker` to build the image, an issue:
```
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='org.apache.hive:hive-standalone-metastore-rest-catalog:4.1.0-SNAPSHOT'}' and 'Vertex{label='org.apache.hive:hive-standalone-metastore-server:4.1.0-SNAPSHOT'}' introduces to cycle in the graph org.apache.hive:hive-standalone-metastore-server:4.1.0-SNAPSHOT --> org.apache.hive:hive-standalone-metastore-rest-catalog:4.1.0-SNAPSHOT --> org.apache.hive:hive-standalone-metastore-server:4.1.0-SNAPSHOT -> [Help 1]